### PR TITLE
Add swagger changes for zone redundancy property to databases, elastic pools and editions

### PR DESCRIPTION
**Pull request by emgu-ms**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by fearthecowboy — APPROVED** [review]
@emgu-ms  -- if you are an MS employee, make sure your account is tied to your domain account so that it's cleared by the CLA check.

if not, you can sign the cla (follow the instructions in the post...)

**Comment by jaredmoo** [comment]
Looks good. Just want to make sure this is on in at least 1 production region and has .NET SDK test written and working on your machine before this gets merged.

**Comment by fearthecowboy** [comment]
@emgu-ms  -- ping this thread when you've satisfied @jaredmoo 's conditions.

File: specification/sql/resource-manager/Microsoft.Sql/2014-04-01/capabilities.json [hunk 1]
```diff
@@ -183,7 +183,8 @@
                 },
                 "zoneRedundant": {
                   "description": "Whether or not zone redundancy is supported for the edition.",
-                  "type": "boolean",
+                  "type": "string",
+                  "format": "boolean"
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/2014-04-01/capabilities.json]: I think you need a comma at the end here

**Review by jaredmoo — APPROVED** [review]
Looks good, please confirm that .NET SDK is working and then let's merge.

**Comment by jaredmoo** [comment]
Oh ho, linter errors. It looks like boolean format doesn't exist. I think you'll have to just remove those format's and treat it as plain string.

**Comment by fearthecowboy** [comment]
Is the value a `boolean`?

swagger uses `"type":"boolean"` -- not `format`

.

**Comment by jaredmoo** [comment]
The value is either `"true"` or `"false"` as a string, not `true` or `false` as a boolean literal. This is an unfortunate limitation in our API stack that implements api-version 2014-04-01. The later api-versions (which are being deployed soon) will have this as a real boolean.

**Comment by fearthecowboy** [comment]
Ah. ok. 

>,< 

Ouch.

**Comment by fearthecowboy** [comment]
Hey @emgu-ms,

are we getting close to being ready to merge this?

**Comment by emgu-ms** [comment]
Yup! We're good! Thanks all :)

**Pull request merged**